### PR TITLE
tickets/dm41106: Updated the documentation in the README.

### PR DESCRIPTION
**Pull request by rbovill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ This repository contains all the interface definition files for all the CSCs of
 
 * (https://github.com/lsst-ts/ts_sal) 
 
-The SAL produces libraries in C++, LabVIEW, Java, PyDDS, and Python, for each CSC.  These libraries are then used by all the CSCs that need to publish or subscribe to any message topic.
+The SAL produces libraries in C++, LabVIEW, and Java for each CSC.  These libraries are then used by all the CSCs that need to publish or subscribe to any message topic.
```
Comment (pkubanek) [inline README.md:19]: This misses ts_salobj, which produces wrappers for Python. Can we somehow include that? Like:

Python object interface is provided by (https://github.com/lsst-ts/ts_salobj).

File: README.md [hunk 2]
```diff
@@ -78,6 +70,10 @@ In case this error occurs, the subsystems list in testutils.py should be updated
```
Comment (pkubanek) [inline README.md:70]: This doesn't include enums and tables directories, which hold constants. Can we include that?

## Constanst

Apart from Python code to manipulate XML definition, the repository contains code with enumeration values and tables used by projects. Those are stored in (https://github.com/lsst-ts/ts_xml/tree/main/python/lsst/ts/xml/enums) and (https://github.com/lsst-ts/ts_xml/tree/main/python/lsst/ts/xml/tables).

File: README.md [hunk 3]
```diff
@@ -78,6 +70,10 @@ In case this error occurs, the subsystems list in testutils.py should be updated
 
 ### Add, Rename or Delete a CSC
 
+**NOTE:** Any changes to the list of CSCs must be approved at the CAP meeting **BEFORE** the changes are made.  Please add an agenda item to the appropriate weekly meeting.
+
+* [Weekly Meeting](https://confluence.lsstcorp.org/display/LSSTCOM/Weekly+Meetings)
+
 Adding, renaming or deleting a CSC, involve operations on files and directories within python/lsst/ts/xml. To add a CSC:
 
 * Add an entry in data/sal_interfaces/SALSubsystems.xml
```
Comment (pkubanek) [inline README.md:79]: This lacks the need to add enums for the new SAL component if XML contains it..

**Review by couger01 — CHANGES_REQUESTED** [review]
I recommend moving each sentence to its own line. In addition, I have a few minor suggestions as well.

File: README.md [hunk 4]
```diff
@@ -78,6 +70,10 @@ In case this error occurs, the subsystems list in testutils.py should be updated
 
 ### Add, Rename or Delete a CSC
 
+**NOTE:** Any changes to the list of CSCs must be approved at the CAP meeting **BEFORE** the changes are made.  Please add an agenda item to the appropriate weekly meeting.
+
+* [Weekly Meeting](https://confluence.lsstcorp.org/display/LSSTCOM/Weekly+Meetings)
+
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply README.md:76]: Referring to pydds would be too specific. The idea of sal is to be an abstraction of what's under it.

[the remark this answers is not in the document]
Comment (wvreeven) [reply README.md:76]: Perhaps `For support of the Python language, see ts_salobj https://github.com/lsst-ts/ts_salobj.`?

File: README.md (hunk 1 above)
Comment (pkubanek) [reply README.md:19]: Sure, it's independent but is used to provide Python binding. This chapter is about bindings, and now completely lacks any mention of how Python is interfaced.

Comment (wvreeven) [reply README.md:19]: The Python binding should not be used anymore and will be removed in the future. So it should not be mentioned and anyone using Python needs to use ts_salobj.

**Review by couger01 — APPROVED** [review]
LGTM.

**Pull request merged**